script.js

```diff
@@ -1,4 +1,109 @@
+// Global States
+var playerTurn = 1; // 1 = Player 1; 2 = Player 2
+var gamePhase = 1; // 1 = Dice Roll; 2 = Choose Order; 3 = Conclusion
+var playerRolls = [];
+// var player1Rolls = [];
+// var player2Rolls = [];
+var playerChoice = 0; // If 1, choose 1st numerical; If 2, choose 2nd numerical
+var finalNumber1 = 0; // Stores final number for Player 1
+var finalNumber2 = 0; // Stores final number for Player 2
+
+// Function: Returns random number from 1 to 6 -inclusive
+var rollDice = function () {
+  var randomDecimal = Math.random() * 6;
+  var randomInteger = Math.floor(randomDecimal);
+  var diceNum = randomInteger + 1;
+  return diceNum;
+};
+
+// Function: Takes user input 1 or 2, to generate combined number's value with chosen 1st numerical
+var chosenOrderNum = function (choice, playerRolls) {
+  if (choice == 1) {
+    chosenNum = Number(String(playerRolls[0]) + String(playerRolls[1]));
+  } else if (choice == 2) {
+    chosenNum = Number(String(playerRolls[1]) + String(playerRolls[0]));
+  }
+  return chosenNum;
+};
+
+// Function: Takes player number input, to return message on what numbers were rolled
+var playerXRolls = function (playerNum) {
+  var index = playerNum - 1;
+  dice1Roll = playerRolls[index][0];
+  dice2Roll = playerRolls[index][1];
+  var message = `You rolled <b>${dice1Roll}</b> for Dice 1 and <b>${dice2Roll}</b> for Dice 2.`;
+  return message;
+};
+
+// Function: Returns error message for not picking 1 or 2.
+var errorMessage = function () {
+  return `Please try again. Enter '1' for Dice 1 or '2' for Dice 2.`;
+};
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  // 1st IF: If game is in Phase 3 (Conclusion)
+  if (gamePhase == 3) {
```

Comment:
generally, its better practice to make your variables like gameState readable by using descriptive words rather than numbers!